app/Helpers/Helpers.php

```diff
@@ -195,9 +195,15 @@ function apiLogs($request, $status_code, $s3_string = false, $ip_address = null)
 
 function generateCacheString($key, $args = [])
 {
-    return strtolower(array_reduce($args, function ($carry, $item) {
+    $cache_string =  strtolower(array_reduce($args, function ($carry, $item) {
         return $carry .= ':' . $item;
     }, $key));
+
+    if (strlen($cache_string) >= 250){
```

Comment:
this incorrectly disallows a cache_string of length 250. the length check should be for > 250

Reply:
I wasn't sure about that, changing it